src/api/Feed.js

```diff
@@ -517,26 +518,45 @@ class Feed extends Base {
         this.errorCallback = errorCallback;
         this.tasksNewAPI = new TasksNewAPI(this.options);
         this.updateFeedItem({ isPending: true }, task.id);
-        this.tasksNewAPI.updateTask({
-            file,
-            task,
-            successCallback: (taskData: Task) => {
-                this.updateFeedItem(
-                    {
-                        ...taskData,
-                        isPending: false,
-                    },
-                    task.id,
+
+        Promise.resolve()
+            .then(() => {
+                return Promise.all(
+                    task.addedAssignees.map(assignee => this.createTaskCollaborator(file, task, assignee)),
```

Comment:
I assume this question has been asked, but is it possible to do this in a batch call? This scheme is going to generate a lot of round-trips to the server.

Reply:
it isn't possible currently :(

Reply:
If anything, the optimization we can consider is `_.chunk(added, 5)` to reduce the number of concurrent requests